votes/populate/export.py

```diff
@@ -93,17 +223,37 @@ def dump_models():
     all_periods.to_parquet(DATA_DIR / "policy_comparison_period.parquet")
 
 
-def improve_votes():
-    div_id_to_key: dict[int, str] = dict(Division.objects.values_list("id", "key"))
+def create_duckdb(quiet: bool = False):
+    """
+    Create a duckdb where all these URLs are available for querying.
+    """
+
+    if not quiet:
+        print("Creating DuckDB with views for all votes data")
+
+    DATA_DIR.mkdir(parents=True, exist_ok=True)
+    duck_db_path = DATA_DIR / "twfy_votes.duckdb"
+
+    # This needs to reference the live site for URLs to connect
+    url_base = "https://votes.theyworkforyou.com/static/data"
```

Comment:
could this not be a setting thing with an env override for local use/testing? Or at least the host name.

Reply:
So it'd probably be a good idea to have that setting around, but it's fiddly. 

The specific problem I had is that duckdb references the URL and does something to check it's valid at view creation - so this requires the server being accessible when running export in development. This is possible but fiddly, but it was less clear to me what this was then testing (while this approach is at least testing 'yep, those are valid parquets on the live site and this duckdb creation works). 

Could also give it the *local* path to validate the local export has created valid parquet files - but that wouldn't then work if you accessed the duckdb through the api.

Reply:
gotcha - yep that's doable.